t/00_cds.t

```diff
@@ -71,4 +71,14 @@ my $myResponse = $xml->parse_string($response->{result});
 $myResponse->removeChild(($myResponse->childNodes())[0]);
 
 is($response->{status_code},0,'Conversion was problem-free.');
-is($myResponse->toString(1),$content_query,'Content query successfully generated');
+
+my @got_lines = split("\n", $myResponse->toString(1));
+my @expected_lines = split("\n", $content_query);
+my $index = 0;
+while (@got_lines) {
+  $index++;
+  my $got_line = shift @got_lines;
+  my $expected_line = shift @expected_lines;
+  is($got_line, $expected_line,'Line $index was different than expected.');
+}
```

Comment:
probably time to add a test utility for the stex tests. Having the failures as line diffs is a million times more readable than dumping the full XML.